packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/provider.ts

```diff
@@ -0,0 +1,48 @@
+import { aws_lambda, aws_iam, Duration } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+/**
+ * Lightweight construct wrapper for custom resource provider 'Custom::AmplifyDynamoDBTable'
+ * Encapsulating the entry point (onEventHandler) within this `Provider` construct is necessary
+ * to maintain a stable custom resource entry point ARN that is consistent with previously deployed
+ * versions when the CDK's provider framework.
```

Comment:
Not sure I follow this last phrase. Did you mean "when the CDK's provider framework was used (to previously deploy)" or something else?

Reply:
Yes, exactly this "when the CDK's provider framework was used (to previously deploy)" 😄

Reply:
Addressed in d22964250b71f4f441cea6229a32df4f4757aeae